src/utils/consent.js

```diff
@@ -27,7 +27,8 @@ export const consentMessages = {
     'consent-message-terms': 'I understand and accept the Terms of Service.',
     'consent-softkeys-policy': 'Policy',
     'consent-softkeys-agree': 'Agree',
-    'consent-softkeys-terms': 'Terms'
+    'consent-softkeys-terms': 'Terms',
+    'consent-privacy-terms': 'Privacy and Terms'
```

Comment:
Added it here instead of en/qqq.json to leave all consent message grouped together

Reply:
This is just for messages that have legal implications and that are translated by legal experts. This is not hooked into TWN so please move this key to en/qqq.